<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grader</class>
 <widget class="QWidget" name="Grader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grader</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Student ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="gradespinner">
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="name_editor"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="print_button">
     <property name="text">
      <string>Print</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="quit_button">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QSlider" name="gradeslider">
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="sliderPosition">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="id_editor"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quit_button</sender>
   <signal>clicked()</signal>
   <receiver>Grader</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gradespinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gradeslider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gradeslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gradespinner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>print()</slot>
 </slots>
</ui>
